### PR TITLE
feat: [PIE-4095]: add git support in clone pipeline

### PR DESCRIPTION
**Pull request by vkbansal**
##### Screenshots:
Screenshot 2022-06-27 at 4 22 59 PM
Screenshot 2022-06-27 at 4 23 07 PM

You can use the following comments to re-trigger PR Checks

- Jest: `retrigger jest`
- Prettier: `retrigger prettier`
- Type Check: `retrigger typecheck`
- ESLint: `retrigger eslint`
- Standards: `retrigger standards`
- Build: `retrigger build`
- Title Check: `retrigger titlecheck`
- Coverage: `retrigger coverage`
- Rebase: `trigger rebase`
- Cypress: `trigger cypress`
- Fix Prettier - `fix prettier`

### CODE REVIEW

**Review by bhavya-sinha — APPROVED** [review]

File: src/modules/70-pipeline/pages/pipelines/views/PipelineCard/PipelineCard.tsx [hunk 1]
```diff
@@ -151,7 +151,6 @@ function ContextMenu({
       <Menu.Item
         icon="duplicate"
         text={getString('projectCard.clone')}
-        disabled={isGitSyncEnabled}
```
Comment (sunnykesh-harness) [inline src/modules/70-pipeline/pages/pipelines/views/PipelineCard/PipelineCard.tsx:154]: Why we removing this? Are we supporting clone even if old gitSync is enabled?

**Review by sunnykesh-harness — APPROVED** [review]

File: src/modules/70-pipeline/pages/pipelines/views/ClonePipelineForm/ClonePipelineForm.test.tsx [hunk 2]
```diff
@@ -81,163 +93,365 @@ jest.mock('services/pipeline-ng', () => ({
...
-    const org = queryByAttribute('name', form, 'destinationConfig.orgIdentifier') as HTMLInputElement
-    const proj = queryByAttribute('name', form, 'destinationConfig.projectIdentifier') as HTMLInputElement
-
-    expect(name.value).toBe('My Pipeline - Clone')
-    expect(id.textContent).toBe('My_Pipeline_Clone')
-    expect(org.value).toBe('Test Org 1')
-    expect(proj.value).toBe('Test Project 1')
-    expect(description.value).toBe(originalPipeline.description)
-  })
+  describe('isGitSimplificationEnabled = false', () => {
```
Comment (ishantmehtaharness) [inline src/modules/70-pipeline/pages/pipelines/views/ClonePipelineForm/ClonePipelineForm.test.tsx:96]: Whats the use case of adding `describe` block inside `describe` block

File: src/modules/70-pipeline/pages/pipelines/views/PipelineCard/PipelineCard.tsx (hunk 1 above)
Comment (ishantmehtaharness) [reply src/modules/70-pipeline/pages/pipelines/views/PipelineCard/PipelineCard.tsx:154]: ➕

Comment (vkbansal) [reply src/modules/70-pipeline/pages/pipelines/views/PipelineCard/PipelineCard.tsx:154]: This was alias for new one only

**Review by ishantmehtaharness — APPROVED** [review]
Looks Good

File: src/modules/70-pipeline/pages/pipelines/views/ClonePipelineForm/ClonePipelineForm.test.tsx (hunk 2 above)
Comment (vkbansal) [reply src/modules/70-pipeline/pages/pipelines/views/ClonePipelineForm/ClonePipelineForm.test.tsx:96]: Logical grouping of test cases

**Pull request merged**